server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -261,8 +275,10 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
                     documentId,
                     tenantId,
                 };
-                logger?.error(`Error fetching deli state from summary`, { messageMetaData });
+                const errorMessage = `Error fetching deli state from summary`;
+                logger?.error(errorMessage, { messageMetaData });
                 logger?.error(JSON.stringify(exception), { messageMetaData });
+                Lumberjack.error(`Error fetching deli state from summary. Exception: ${JSON.stringify(exception)}`);
```

Comment:
1. Let's add documentId and tenantId to this Lumberjack log.
2. Let's extract the error from the message and provide it as a the `exception` parameter.